UI/MailerUI/UIxMailFolderActions.m

```diff
@@ -106,7 +106,7 @@ - (WOResponse *) renameFolderAction
   keyForMsgUIDs = [NSString stringWithFormat:@"/%@/%@", currentAccount, currentMailbox];
 
   newFolderName = [[context request] formValueForKey: @"name"];
-  newKeyForMsgUIDs = [[NSString stringWithFormat:@"/%@/folder%@", currentAccount, newFolderName] asCSSIdentifier];
+  newKeyForMsgUIDs = [NSString stringWithFormat:@"/%@/folder%@", [currentAccount asCSSIdentifier], [newFolderName asCSSIdentifier]];
```

Comment:
is there any difference here that I'm missing? `[@"folder" asCSSIdentifier] == @"folder"` so a concatenation of already `asCSSIdentifier` strings should be the same as concatenation of strings and then apply `asCSSIdentifier`